gtfs/spec/en/reference.md

```diff
@@ -612,6 +624,69 @@ Describes levels in a station. Useful in conjunction with `pathways.txt`, and is
 |  `level_index` | Float | **Required** | Numeric index of the level that indicates its relative position. <br><br>Ground level should have index `0`, with levels above ground indicated by positive indices and levels below ground by negative indices.|
 |  `level_name` | Text | Optional | Name of the level as seen by the rider inside the building or station.<hr>_Example: Take the elevator to "Mezzanine" or "Platform" or "-1"._|
 
+### location_groups.txt
+
+File: **Optional**
+
+Primary key (`location_group_id`, `location_id`)
+
+Defines location groups that indicate groups of stops where a rider may request pickup or drop off.
```

Comment:
I agree with @abyrd's wording change suggestion with @tzujenchanmbd's adjustment of "in on-demand service." "location_group" was the original name because one could include geojson locations as well as stops. I agree it's not perfect, but I prefer keeping it as-is for the following reasons:
1. The word "location" has broader application which is convenient if there is potential to add other "location" types to the file if a use case is discovered.
2. I don't think "location" is inaccurate for a file that only references stops either; in the spec, stops are called "locations" and even have associated `location_types`.
3.  In my opinion, including the word "stop" in the file name causes more confusion considering there are more instances of that word elsewhere in the spec: `stop_areas.txt`, `stop_times.txt`, `stops.txt`, not to mention all the field names with the word "stop." `stop_areas.txt` is an especially important case because it actually has the same relational function as location_groups, only for a different component of the spec (fares). Imagine having two files named `stop_areas.txt` and `stop_groups.txt` that are **nearly functionally identical**, but meant for different purposes.

Reply:
> Not sure about `flex_stop_groups` since we currently don't use the term "flex" in the spec proposal. 
See comment below:
https://github.com/google/transit/pull/388#discussion_r1419896819
I am not insisting on the specific term "flex" (though it's short and distinct), just pointing out that prefixing/namespacing entity types that reference each other within a single coherent spec extension could be very useful.